File: Modul 6/Soal 5/PRAK505-2210817310006-Ahmad Muhsin Zein.py
def Biodata (tahunLahir, A, B) :
    tahun_sekarang = 2020
    umur = tahun_sekarang - tahunLahir
    print('\nPerkenalkan Nama Saya : '+ A)
    print("Umur Saya : %d" %(umur))
    print('Saya adalah Angkatan : ', tahun_sekarang)
    print('Asal Saya dari : '+ B)

tahunLahir = int(input())
A = str(input())
B = str(input())
Biodata(tahunLahir, A, B)

File: Modul 6/Soal 5/PRAK505-2210817310006-Ahmad Muhsin Zein.c
#include <stdio.h>

void biodata(int tahunLahir, char *Namaku, char *Asal){
    int tahun_sekarang = 2020;
    printf("\nPerkenalkan Nama Saya : %s\n", Namaku);
    printf("Umur Saya : %d\n", tahun_sekarang - tahunLahir);
    printf("Saya Adalah Angkatan : %d\n", tahun_sekarang);
    printf("Asal Saya dari : %s", Asal);
}
int main()
{
    int tahunLahir;
    char Namaku[20],Asal[15];
    scanf("%d", &tahunLahir);
    scanf(" %[^\n]%*c",&Namaku );
    scanf(" %[^\n]%*c",&Asal );
    biodata(tahunLahir,Namaku,Asal);
    return 0;
}
